daphne_worker/src/durable/mod.rs

```diff
@@ -35,14 +36,27 @@ const ERR_NO_VALUE: &str = "No such value in storage.";
 // TODO(bhalley) does this need to be configurable?
 const MAX_KEYS: usize = 128;
 
+const DEFAULT_RETRY_INTERVAL: Duration = Duration::from_millis(500);
```

Comment:
I would decrease it to 100ms. eyeball workers don't have much CPU / time to spare.

Reply:
Done.